### ISSUE TITLE
Hotmail does not work (Recipients belong to multiple regions)

### ISSUE THREAD

**Issue by yaiqsa** [reacherhq/check-if-email-exists#1334]
### Email Provider

hotmail.nl, hotmail.com

### Version of check-if-email-exists (if running it yourself)

0.9.1

### What happened?

I executed this from the server that I usually send mail from:
(I replaced my actual mailing domain etc. with mocks in the command & logs)
```
RUST_LOG=debug ./check_if_email_exists --from-email test@domain.tld --hello-name domain.tdl user@hotmail.nl
```

This should return a "safe", because the hotmail address I used exists.

The thing is; when I manually open a telnet session to the outlook mailserver, and I pass the actual mail address, it is accepted. When I emulate Reacher and first try a random receiver address, and then pass my actual mail address in the same SMTP session, I get the same `Recipients belong to multiple regions` error. So I think the problem is that Reacher's way of checking the address just doesn't work properly for outlook. Maybe it should use two separate sessions?

### Relevant log output

```shell
[2023-09-19T12:20:23Z DEBUG reacher] [email=user@hotmail.nl] Checking email "user@hotmail.nl"
[2023-09-19T12:20:23Z DEBUG reacher] [email=user@hotmail.nl] Found the following syntax validation: SyntaxDetails { address: Some(EmailAddress("user@hotmail.nl")), domain: "hotmail.nl", is_valid_syntax: true, username: "user", normalized_email: Some("user@hotmail.nl"), suggestion: None }
[2023-09-19T12:20:23Z DEBUG trust_dns_proto::xfer::dns_handle] querying: hotmail.nl MX
[2023-09-19T12:20:23Z DEBUG trust_dns_resolver::name_server::name_server_pool] sending request: [Query { name: Name("hotmail.nl"), query_type: MX, query_class: IN }]
[2023-09-19T12:20:23Z DEBUG trust_dns_resolver::name_server::name_server] reconnecting: NameServerConfig { socket_addr: 127.0.0.53:53, protocol: Udp, tls_dns_name: None, trust_nx_responses: false, bind_addr: None }
[2023-09-19T12:20:23Z DEBUG trust_dns_proto::xfer] enqueueing message: [Query { name: Name("hotmail.nl"), query_type: MX, query_class: IN }]
[2023-09-19T12:20:23Z DEBUG trust_dns_proto::udp::udp_stream] created socket successfully
[2023-09-19T12:20:23Z DEBUG trust_dns_proto::udp::udp_client_stream] received message id: 9466
[2023-09-19T12:20:23Z DEBUG trust_dns_resolver::error] Response:9466:RD,RA:NoError:1/0/2
[2023-09-19T12:20:23Z DEBUG trust_dns_resolver::error] Response:9466:RD,RA:NoError:1/0/2
[2023-09-19T12:20:23Z DEBUG reacher] [email=user@hotmail.nl] Found the following MX hosts: ["10 eur.olc.protection.outlook.com."]
[2023-09-19T12:20:23Z DEBUG reacher] [email=user@hotmail.nl] Found the following misc details: MiscDetails { is_disposable: false, is_role_account: false, gravatar_url: None, haveibeenpwned: None }
[2023-09-19T12:20:23Z DEBUG reacher] [email=user@hotmail.nl] Check SMTP [attempt=1] on [host=eur.olc.protection.outlook.com.:25]
[2023-09-19T12:20:23Z DEBUG trust_dns_proto::xfer::dns_exchange] io_stream is done, shutting down
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 220 DB8EUR05FT023.mail.protection.outlook.com Microsoft ESMTP MAIL Service ready at Tue, 19 Sep 2023 12:20:22 +0000<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::smtp_client] connection established to eur.olc.protection.outlook.com:25
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] >> EHLO domain.tld<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-STARTTLS<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-STARTTLS<CRLF>250-8BITMIME<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-STARTTLS<CRLF>250-8BITMIME<CRLF>250-BINARYMIME<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-STARTTLS<CRLF>250-8BITMIME<CRLF>250-BINARYMIME<CRLF>250-CHUNKING<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-STARTTLS<CRLF>250-8BITMIME<CRLF>250-BINARYMIME<CRLF>250-CHUNKING<CRLF>250 SMTPUTF8<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::smtp_client] server DB8EUR05FT023.mail.protection.outlook.com with {SmtpUtfEight, Pipelining, EightBitMime, StartTls}
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] >> STARTTLS<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 220 2.0.0 SMTP server ready<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::smtp_client] connection encrypted
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] >> EHLO domain.tld<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-8BITMIME<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-8BITMIME<CRLF>250-BINARYMIME<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-8BITMIME<CRLF>250-BINARYMIME<CRLF>250-CHUNKING<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT023.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-8BITMIME<CRLF>250-BINARYMIME<CRLF>250-CHUNKING<CRLF>250 SMTPUTF8<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::smtp_client] server DB8EUR05FT023.mail.protection.outlook.com with {EightBitMime, SmtpUtfEight, Pipelining}
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] >> MAIL FROM:<test@domain.tld><CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250 2.1.0 Sender OK<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] >> RCPT TO:<Zl2ieK3Pwtmp99y@hotmail.nl><CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 550 5.5.0 Requested action not taken: mailbox unavailable (S2017062302). [DB8EUR05FT023.eop-eur05.prod.protection.outlook.com 2023-09-19T12:20:23.826Z 08DBB85E9F29E5D3]<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] >> RCPT TO:<user@hotmail.nl><CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 452 4.5.3 Recipients belong to multiple regions ATTR38 [DB8EUR05FT023.eop-eur05.prod.protection.outlook.com 2023-09-19T12:20:23.857Z 08DBB85E9F29E5D3]<CRLF>
[2023-09-19T12:20:23Z DEBUG reacher] [email=user@hotmail.nl] Got result for [attempt=1] on [host=eur.olc.protection.outlook.com.:25], [result=Err(SmtpError(Transient(Response { code: Code { severity: TransientNegativeCompletion, category: MailSystem, detail: Two }, message: ["4.5.3 Recipients belong to multiple regions ATTR38 [DB8EUR05FT023.eop-eur05.prod.protection.outlook.com 2023-09-19T12:20:23.857Z 08DBB85E9F29E5D3]"] })))]
[2023-09-19T12:20:23Z DEBUG reacher] [email=user@hotmail.nl] Potential greylisting detected, retrying.
[2023-09-19T12:20:23Z DEBUG reacher] [email=user@hotmail.nl] Check SMTP [attempt=2] on [host=eur.olc.protection.outlook.com.:25]
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 220 DB8EUR05FT022.mail.protection.outlook.com Microsoft ESMTP MAIL Service ready at Tue, 19 Sep 2023 12:20:22 +0000<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::smtp_client] connection established to eur.olc.protection.outlook.com:25
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] >> EHLO domain.tld<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-STARTTLS<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-STARTTLS<CRLF>250-8BITMIME<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-STARTTLS<CRLF>250-8BITMIME<CRLF>250-BINARYMIME<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-STARTTLS<CRLF>250-8BITMIME<CRLF>250-BINARYMIME<CRLF>250-CHUNKING<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-STARTTLS<CRLF>250-8BITMIME<CRLF>250-BINARYMIME<CRLF>250-CHUNKING<CRLF>250 SMTPUTF8<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::smtp_client] server DB8EUR05FT022.mail.protection.outlook.com with {Pipelining, SmtpUtfEight, EightBitMime, StartTls}
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] >> STARTTLS<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 220 2.0.0 SMTP server ready<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::smtp_client] connection encrypted
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] >> EHLO domain.tld<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-8BITMIME<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-8BITMIME<CRLF>250-BINARYMIME<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-8BITMIME<CRLF>250-BINARYMIME<CRLF>250-CHUNKING<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250-DB8EUR05FT022.mail.protection.outlook.com Hello [1.2.3.4]<CRLF>250-SIZE 49283072<CRLF>250-PIPELINING<CRLF>250-DSN<CRLF>250-ENHANCEDSTATUSCODES<CRLF>250-8BITMIME<CRLF>250-BINARYMIME<CRLF>250-CHUNKING<CRLF>250 SMTPUTF8<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::smtp_client] server DB8EUR05FT022.mail.protection.outlook.com with {SmtpUtfEight, Pipelining, EightBitMime}
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] >> MAIL FROM:<test@domain.tld><CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 250 2.1.0 Sender OK<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] >> RCPT TO:<VVLBEDLqENTcl5k@hotmail.nl><CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 550 5.5.0 Requested action not taken: mailbox unavailable (S2017062302). [DB8EUR05FT022.eop-eur05.prod.protection.outlook.com 2023-09-19T12:20:24.061Z 08DBB85C6F3051DC]<CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] >> RCPT TO:<user@hotmail.nl><CRLF>
[2023-09-19T12:20:23Z DEBUG async_smtp::smtp::client::inner] << 452 4.5.3 Recipients belong to multiple regions ATTR38 [DB8EUR05FT022.eop-eur05.prod.protection.outlook.com 2023-09-19T12:20:24.092Z 08DBB85C6F3051DC]<CRLF>
[2023-09-19T12:20:23Z DEBUG reacher] [email=user@hotmail.nl] Got result for [attempt=2] on [host=eur.olc.protection.outlook.com.:25], [result=Err(SmtpError(Transient(Response { code: Code { severity: TransientNegativeCompletion, category: MailSystem, detail: Two }, message: ["4.5.3 Recipients belong to multiple regions ATTR38 [DB8EUR05FT022.eop-eur05.prod.protection.outlook.com 2023-09-19T12:20:24.092Z 08DBB85C6F3051DC]"] })))]
{
  "input": "user@hotmail.nl",
  "is_reachable": "unknown",
  "misc": {
    "is_disposable": false,
    "is_role_account": false,
    "gravatar_url": null,
    "haveibeenpwned": null
  },
  "mx": {
    "accepts_mail": true,
    "records": [
      "eur.olc.protection.outlook.com."
    ]
  },
  "smtp": {
    "error": {
      "type": "SmtpError",
      "message": "transient: 4.5.3 Recipients belong to multiple regions ATTR38 [DB8EUR05FT022.eop-eur05.prod.protection.outlook.com 2023-09-19T12:20:24.092Z 08DBB85C6F3051DC]"
    }
  },
  "syntax": {
    "address": "user@hotmail.nl",
    "domain": "hotmail.nl",
    "is_valid_syntax": true,
    "username": "user",
    "normalized_email": "user@hotmail.nl",
    "suggestion": null
  }
}
```

**Comment by rbustos** [comment]
Seems it happens with all *mail.protection.outlook.com related domains.

I got same results with valid and invalid (no existing email) in our tests

`DEBUG async_smtp::smtp::client::inner] << 451 4.4.4 Temporary server error. Please try again later ATTR1 [MN1PEPF0000F0E2.namprd04.prod.outlook.com 2023-09-23T03:53:11.518Z 08DBBB8502D01A60]<CRLF>`

**Comment by amaury1729** [comment]
In https://github.com/reacherhq/check-if-email-exists/pull/1336, I removed the first random email check for hotmail (and gmail and yahoo). This should fix this issue, I'll tag 0.9.1 in the coming days.
